part2/lists.js

```diff
@@ -0,0 +1,59 @@
+document.body.style.fontFamily="Arial, sans-serif";
+
+document.querySelectorAll("li")[0].innerText += " Chandler Bing ";
+document.querySelectorAll("li")[1].innerText += " Duck & chiken";
+document.querySelectorAll("li")[2].innerText += " Central perk ";
+
+const allItems = document.querySelectorAll('li');
+for(let i = 0; i < allItems.length; i ++){
+allItems[i].className = ".listitem";

```

Comment:
You can remove the . here
```suggestion
allItems[i].className = "listitem";
```